config/software/stunnel.rb

```diff
@@ -53,16 +53,16 @@
     # an env variable to redirect it to the correct location
     env["WIN32_SSL_DIR_PATCHED"] = "#{install_dir}/embedded"
 
-    target = windows_arch_i386? ? "mingw" : "mingw64"
-    make target, env: env, cwd: "#{project_dir}/src"
-
+    mingw = ENV["MSYSTEM"].downcase
```

Comment:
Looking at the ChefDK builder environment, it appears that the 32 bit build systems export `MSYSTEM=MINGW32`.

I'm 👍 for using the environment variable, but because stunnel only has make targets for `mingw` and `mingw64` can you go back to a ternary to determine the correct make target?